db/migrate/20230904180116_create_users.rb

```diff
@@ -0,0 +1,9 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :email, null: false, default: ""
```

Comment:
- [ ] As per Entity Relationship diagram, the column in the User table is `name`. But I see that you are using `email`. This needs to be fixed

Reply:
Thank you for the feed back. It is fixed.